reflect_test.go

```diff
@@ -106,6 +106,10 @@ type TestUser struct {
 	Priorities []int     `json:"priorities,omitempty" jsonschema:"enum=-1,enum=0,enum=1,enun=2"`
 	Offsets    []float64 `json:"offsets,omitempty" jsonschema:"enum=1.570796,enum=3.141592,enum=6.283185"`
 
+	// Tests for deprecated
+	DeprecatedField string   `json:"deprecated_field" jsonschema:"deprecated=true"`
+	DeprecatedSlice []string `json:"deprecated_slice" jsonschema:"deprecated=true"`
```

Comment:
I find it surprising that you have to add `=true` here. When would anybody ever set this to false? I would have expected that you just say `jsonschema:"deprecated"`, like you do e.g. for `json:"omitempty"`.